labconnect/main/routes.py

```diff
@@ -60,19 +60,24 @@ def profile():
 
     return result
 
+@main_blueprint.route("/departments", methods=["GET"])
+def departmentCards():
+    results = []
 
-@main_blueprint.route("/department")
-def department():
+    data = db.session.execute(db.select(RPIDepartments)).scalars()
```

Comment:
optional: select specific columns like name, school_id, and image then serialize the response instead of creating dictionaries